src/voice/Playback.ts

```diff
@@ -92,6 +95,14 @@ export class Playback extends EventEmitter implements IDestroyable {
         return this.resampledWaveform;
     }
 
+    /**
+     * Stable waveform for representing a thumbnail of the media. Values are
+     * guaranteed to be between zero and one, inclusive.
+     */
+    public get waveformThumbnail(): number[] {
+        return this.thumbnailWaveform;
+    }
```

Comment:
this is only ever read, we could make it `public readonly` and have TS guard it instead

Reply:
doh, of course. I'm honestly impressed at my naming here managing to avoid the conflict that would make it obvious.

Reply:
At first, looking at the other file I was like wait that is totally backwards before I saw the getter lol